### ISSUE TITLE
./mconfig outputs `E: Not inside a git repository and no VERSION file found. Abort.`

### ISSUE THREAD

**Issue by panda1100** [singularityware/singularity#6225]
### Version of Singularity:

```
singularity version 3.8.3
```

### Expected behavior

configure singularity-3.8.3.tar.gz by `./mconfig` , configuration runs without problem.

### Actual behavior

`./mconfig` outputs `E: Not inside a git repository and no VERSION file found. Abort.`
then stopped.

### Steps to reproduce this behavior

```
wget https://github.com/hpcng/singularity/releases/download/v3.8.3/singularity-3.8.3.tar.gz
tar zxcf  singularity-3.8.3.tar.gz
cd singularity-3.8.3
./mconfig
```
If I create VERSION file with command below, it resolved problem.

```
echo 3.8.3 >> VERSION
```

### What OS/distro are you running

```
NAME="Ubuntu"
VERSION="20.04.2 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.2 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```
and my go version

```
go version go1.16.4 linux/amd64
```

**Comment by anChaOs** [comment]
I also encountered the same problem when I installed version 3.8.7.

what I downloaded is: `[Source code (tar.gz)](https://github.com/apptainer/singularity/archive/refs/tags/v3.8.7.tar.gz)`

however, the problem fixed when I download: `[singularity-3.8.7.tar.gz](https://github.com/apptainer/singularity/releases/download/v3.8.7/singularity-3.8.7.tar.gz)`

**Comment by kmuriki** [comment]
Singularity repo is now retired as the code base is now moved to Apptainer. We are closing all the old issues under the old Singularity repo. For further assistance please open a new issue under the new Apptainer repo. Thanks for your support.
